data_model/src/expression.rs

```diff
@@ -668,8 +833,20 @@ impl From<If> for ExpressionBox {
 /// `Contains` expression.
 /// Returns `true` if `collection` contains an `element`, `false` otherwise.
 #[derive(
-    Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema, PartialOrd, Ord,
+    Debug,
+    Display,
+    Clone,
+    PartialEq,
+    Eq,
+    Decode,
+    Encode,
+    Deserialize,
+    Serialize,
+    IntoSchema,
+    PartialOrd,
+    Ord,
 )]
+#[display(fmt = "`{}` contains `{}`", collection, element)]
```

Comment:
```suggestion
#[display(fmt = "{}.contains({})", "collection.parenthesise(Operation::negation)", element)]
```
Parenthesise everything except single objects for collection. Use familiar syntax for Rust developers. Other libraries will present this differently.

Reply:
I think, it's better to use `MethodCall` operation here instead of `Negation`

Reply:
Yeah, because we have more than just contains.